models/public/yolo-v4-tf/accuracy-check.yml

```diff
@@ -41,7 +41,7 @@ models:
             apply_to: prediction
             min_confidence: 0.001
             remove_filtered: true
-          - type: nms
+          - type: diou_nms
```

Comment:
@eizamaliev could you please comment, why during adding yolo-v4 you do not use diou_nms?
@playboy007gy I want to note that we do not have intention to get the most accurate results that possible, our purposes to reproduce original testing condition and as I understand this posprocessing used only during training, final result in the article provided without it.

Reply:
In the original paper, diou was used for training and inference.
(The purpose of inference is to show the ability gained from training, so the same algorithm as the training condition must be used)
And we want to reproduce the original testing conditions, I think diou is also necessary.

Reply:
@eaidova Yes, indeed. They add diou nms sorting in [commint](https://github.com/AlexeyAB/darknet/commit/c7e3e2ee9ec2d8fff447d83736e13bce0938015f).